sphinx_immaterial/apidoc_formatting.py

```diff
@@ -134,6 +134,7 @@ def visit_desc(self, node: sphinx.addnodes.desc) -> None:
 
         # Augment the list of classes with `objdesc` to make it easier to
         # style these without resorting to hacks.
+        # add highlight to invoke syntax highlighting in CSS
```

Comment:
this comment belongs in the `visit_desc_signature` method below instead.

Reply:
This placement is actually a artifact from local dev testing. Good catch!